### PR TITLE
completed module_7

### PR DESCRIPTION
**Pull request by vladSHOKO**
@Lehersun module 7 ready to check

### CODE REVIEW

File: module_7/index.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<?
```
Comment (Lehersun) [inline module_7/index.php]: it will not work without key word `php` it should looks like <?php

File: module_7/templates/task_3_card.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <div class="text-xl border-b-2 pb-4">
+        Исследуемое значение: <b><?=var_export($value)?></b>, тип <b><?=gettype($value)?></b>
+    </div>
+    <ul class="space-y-2 mt-4 list-disc list-inside text-sm text-gray-700">
+
+        <li>Если привести его к целому число, то оно равно 1: <b class="text-black text-base"><?=((int) $value) === 1 ? 'Да' : 'Нет'?></b></li>
+        <li>Если привести его к boolean, то оно равно true: <b class="text-black text-base"><?=((boolean) $value) === true ? 'Да' : 'Нет'?></b></li>
+        <li>Если привести его к string, то оно равно пустой строке: <b class="text-black text-base"><?=((string) $value) === "" ? 'Да' : 'Нет'?></b></li>
+        <li>Если привести его к string, то оно равно '1': <b class="text-black text-base"><?=((string) $value) === '1' ? 'Да' : 'Нет'?></b></li>
+
```
Comment (Lehersun) [inline module_7/templates/task_3_card.php]: All works fine, but page contains php warnings when trying to convert array to string. You can avoid those warnings with `error_reporting(16);
` command placed before this code

**Review by Lehersun — CHANGES_REQUESTED** [review]

**Review by vladSHOKO — COMMENTED** [review]
@Lehersun corrected task ready to check. Was remade codestyle and corrected erros in convertation array to string

**Review by vladSHOKO — COMMENTED** [review]
@Lehersun code style remade

**Pull request merged**